src/fixtures/data/admission-create-clusterrole.json

```
@@ -0,0 +1,64 @@
+{
+  "uid": "2ac28f03-c045-4af6-86f1-aa0007571863",
+  "kind": {
+    "group": "rbac.authorization.k8s.io",
+    "version": "v1",
+    "kind": "ClusterRole"
+  },
+  "resource": {
+    "group": "rbac.authorization.k8s.io",
+    "version": "v1",
+    "resource": "clusterroles"
+  },
+  "requestKind": {
+    "group": "rbac.authorization.k8s.io",
+    "version": "v1",
+    "kind": "ClusterRole"
+  },
+  "requestResource": {
+    "group": "rbac.authorization.k8s.io",
+    "version": "v1",
+    "resource": "clusterroles"
+  },
+  "name": "pod-creator",
+  "operation": "CREATE",
+  "userInfo": {
+    "username": "system:admin",
+    "groups": ["system:masters", "system:authenticated"]
+  },
+  "object": {
+    "kind": "ClusterRole",
+    "apiVersion": "rbac.authorization.k8s.io/v1",
+    "metadata": {
+      "name": "pod-creator",
+      "creationTimestamp": null,
+      "managedFields": [
+        {
+          "manager": "kubectl-create",
+          "operation": "Update",
+          "apiVersion": "rbac.authorization.k8s.io/v1",
+          "time": "2024-11-08T16:39:03Z",
+          "fieldsType": "FieldsV1",
+          "fieldsV1": {
+            "f:rules": {}
+          }
+        }
+      ]
```

```suggestion
      "managedFields": [
        {
          "manager": "kubectl-create",
          "operation": "Update",
          "apiVersion": "rbac.authorization.k8s.io/v1",
          "time": "2024-11-08T16:39:03Z",
          "fieldsType": "FieldsV1",
          "fieldsV1": {
            "f:rules": {}
          }
        }
      ]
```
I think these are just noise -- any chance we can cut the `managedFields` out?  Our previous data fixtures didn't include them either.